react/features/base/ui/components/web/MultiSelect.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useCallback, useMemo, useRef } from 'react';
+import { makeStyles } from 'tss-react/mui';
+
+import { IconCloseLarge } from '../../../icons/svg';
+import Popover from '../../../popover/components/Popover.web';
+import { MultiSelectItem } from '../types';
+
+import ClickableIcon from './ClickableIcon';
+import ContextMenu from './ContextMenu';
+import ContextMenuItem from './ContextMenuItem';
+import ContextMenuItemGroup from './ContextMenuItemGroup';
+import Input from './Input';
+
+type Props = {
+    autoFocus?: boolean;
+    disabled?: boolean;
+    filterValue?: string;
+    hideList: Function;
+    isOpen?: boolean;
+    items: MultiSelectItem[];
+    noMatchesText?: string;
+    onFilterChange?: (value: string) => void;
+    onRemoved: (item: any) => void;
+    onSelected: (item: any) => void;
+    placeholder?: string;
+    selectedItems?: MultiSelectItem[];
+};
+
+const useStyles = makeStyles()(theme => {
+    return {
+        contextMenu: {
+            position: 'relative' as const,
```

Comment:
`as const` should only be necessary when using `withStyles`. For `makeStyles` it shouldn't be necessary